oai-pmh-handler/src/test/resources/publication.context

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
I guess this will be a cause of great sadness at some point

Reply:
It will be removed from here once we take charge and create a simplified internal model for the search document.